src/Http/ApiClient.php

```diff
@@ -21,6 +21,15 @@
  */
 class ApiClient
 {
+    /**
+     * Error constants.
+     */
+    const ERROR_ENVIRONMENT_TYPE_UNKNOWN = 'Unknown environmentType "%s"';
+    const ERROR_MAC_OS_CURL_VERSION = 'Your PHP seems to be linked to the MacOS provided curl binary. ' .
+    'This is incompatible with our SDK, please reinstall by running: "brew reinstall %s --with-homebrew-curl".%s';
+    const ERROR_SANDBOX_DOES_NOT_SUPPORT_PINNED_KEY =
```

Comment:
Shouldn't "key" be plural?

Reply:
It's either 'Sandbox does not support pinned keys' (in general) or 'Sandbox does not support the usage of a pinned key' (singular).

Reply:
I see.